docs/www/guide-wasm-elastic.md

```diff
@@ -1,261 +1,283 @@
-within `rpk`.
+Here's the real fun -- seeing the transform work.
+
+1. To transform data, produce records to the `origin` topic.
 
->Note: we need to publish the records with `logService:elastic` 
->headers in order to those records pass the `containHeader` 
->validation into `processRecord` function on our wasm script.  
+    ```bash
+    $ rpk topic produce origin -H logService:elastic -n 5
+    ```
 
-```bash
-$ rpk topic produce origin -H logService:elastic -n 5
-```
+    We publish the records with `logService:elastic`
+    because they are required by our Wasm script to pass the `containHeader` validation
+    and go to the `processRecord` function on our Wasm script.
 
-In `/var/lib/redpanda/coprocessor/logs/wasm` you should see log line that look like [this]
+    You can see a log line in `/var/lib/redpanda/coprocessor/logs/wasm` that look like:
 
-```
-Applying Wasm process function
-Index record value to Elasticsearch
-```
+    ```
+    Applying Wasm process function
+    Index record value to Elasticsearch
+    ```
 
 ## Verify Elasticsearch results
-Now we need to validate that our Wasm script, should take records 
-and publish them to Elasticsearch. To do this, we can use the same 
-Elasticsearch client for making a query to Elasticsearch, which 
-query should take every hist that has a  `result_wasm` index.
-
-Will create `elastic-query.js`.
-
-```js
-const { Client } = require("@elastic/elasticsearch")
-
-const client = new Client({ node: 'http://localhost:9200' })
-
-client.search({
-  index: 'result_wasm'
-}).then((elasticResult) => {
-  const results = elasticResult.body.hits.hits.map(record => record._source)
-  console.log(results)
-})
-```
- expected output:
-
- ```bash
- $ node elastic-query.js
- [
-  { record_message: 'Information from Redpanda + Wasm' },
-  { record_message: 'Information from Redpanda + Wasm' },
-  { record_message: 'Information from Redpanda + Wasm' },
-  { record_message: 'Information from Redpanda + Wasm' },
-  { record_message: 'Information from Redpanda + Wasm' }
-]
- ```
-
-We should see 5 items in the list, because we send 5 records when 
-we use `$ rpk topic produce origin -H logService:elastic -n 5`, `-n` is the
-number of record that we are going to publish on `origin` topic. 
-
-Now you have a Wasm script with Elasticsearch connection, this is a 
-basic example about how to take advantage of Redpanda and Wasm 
-engine, try it yourself and share your knowledge.
\ No newline at end of file
+
+Now we need to validate that our Wasm script received the records and published them to Elasticsearch.
+To do this, we'll create a query to Elasticsearch that returns every hit that has a  `result_wasm` index.
+
+1. Save this query in a file names `elastic-query.js`:
+
+    ```js
+    const { Client } = require("@elastic/elasticsearch")
+
+    const client = new Client({ node: 'http://localhost:9200' })
+
+    client.search({
+      index: 'result_wasm'
```

Comment:
could you add `size: 1000`? please, the default value is 25, it could be small value for our users